helm/prometheus-rules/values.yaml

```diff
@@ -2,8 +2,8 @@ name: prometheus-rules
 namespace: monitoring
 serviceType: managed
 project:
-  branch: '[[ .Branch ]]'
-  commit: '[[ .SHA ]]'
+  branch: 'capi-checks'
```

Comment:
I think you want to revert that once you're done. 😉

Reply:
Good catch, thanks ❤️